src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -897,6 +912,9 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
         {templateID &&
           HAS_REACTIONS_TEMPLATES.includes(templateID) &&
           renderShowReactions()}
+        {templateID &&
+          HAS_REACTIONS_TEMPLATES.includes(templateID) &&
+          renderShowShoutouts()}
```

Comment:
i suppose we can render both `renderShowShoutouts() and renderShowReactions()` after running the `templateID && HAS_REACTIONS_TEMPLATES.includes(templateID)`  check once

Reply:
The logic to do so may end up making it look more convoluted than it’s worth to save a Boolean check.